libraries/base/GHC/IO/Encoding/Iconv.hs

```diff
@@ -58,12 +59,12 @@ iconv_trace s
 
 {-# NOINLINE localeEncodingName #-}
```

Comment:
@zjhmale The original implementation in GHC's base had NOINLINE because it used unsafePerformIO. We will eventually use that implementation once the equivalent of c_localeEncoding is implemented.